frontend/src/features/mapSideBar/hooks/useQueryParamSideBar.tsx

```diff
@@ -21,14 +22,20 @@ const useQueryParamSideBar = (): IMapSideBar => {
   const history = useHistory();
 
   const searchParams = queryString.parse(location.search);
-  useEffect(() => {
+  useDeepCompareEffect(() => {
```

Comment:
Why the change here?

Reply:
useEffect wasn't picking up changes within the object dependency array since the reference was the same.